From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: derklaro <git@derklaro.dev>
Date: Mon, 20 Jun 2022 17:06:49 +0200
Subject: [PATCH] disable forge module due to include-build remapping issues


diff --git a/settings.gradle b/settings.gradle
index 82a69d0eef047ce1e636c64a65ec6248a41ffd89..095042e0f55fb1112bb0a2870b00d015006c48ac 100644
--- a/settings.gradle
+++ b/settings.gradle
@@ -22,7 +22,7 @@ include (
         'spark-velocity4',
         'spark-sponge7',
         'spark-sponge8',
-        'spark-forge',
+//        'spark-forge', cloudnet - disabled due to remapping issues
         'spark-fabric',
         'spark-nukkit',
         'spark-waterdog',
